### DISCUSSION TITLE
form-label vs col-form-label

### DISCUSSION THREAD

**Discussion by stijn7621** [twbs/bootstrap#39276] [Q&A] [answered]
Hello, 

I'm a bit confused whether to use the form-label versus col-form-label.

**Comment by julien-deramond** [comment, marked as answer]
Hello @stijn7621 
This is linked to the orientation of your form.
* For [horizontal forms](https://getbootstrap.com/docs/5.3/forms/layout/#horizontal-form), each row (with `.row`) contains the label and a form control, so the label needs a `.col-form-label` to have the right padding in order to be vertically aligned to the form control.
* For [vertical forms (by default in our doc)](https://getbootstrap.com/docs/5.3/forms/layout/#forms), `.form-label` is used.
